### ISSUE TITLE
Any way to use passport-azure-ad without sessions/cookies

### ISSUE THREAD

**Issue by BillyCottrell** [AzureAD/microsoft-authentication-library-for-js#3424]
Hi I have checked issue AzureAD/passport-azure-ad#343, and I am kind of in the same situation I am using angular 9 for the website and loopback with express middleware for the backend. Thing is I have my own auth setup which works without issues, and I want to use Microsoft Graph to get booking information. So I don't need user information or anything else, except for the access tokens. Is there a way to be able to set this up? I have tried using the Microsoft Graph demo to see how things work, but they work with all the auth stuff and user information that is not necessary in my case.

**Comment by BillyCottrell** [comment]
My apologies for the late response, but didn't notice the update + the project was placed on hold for a long time. As a matter of fact I just started to look back into this and I have to mention that I kept the code for the cookies and session, I just integrated it with my API to make it work. I probably could have removed parts of it but didn't because I didn't want to mess with it. Any ways I'm going to redo the code soon, as I am going to make a different application that will use this again.
